tests/integration/update_cluster/launch_templates/in-v1alpha2.yaml

```diff
@@ -69,10 +69,11 @@ spec:
   minSize: 2
   role: Node
   instanceProtection: true
+  maxPrice: "0.1"
+  spotDurationInMinutes: 120
   subnets:
   - us-test-1b
 ---
-
```

Comment:
Please add back this line to reduce the diff.

Reply:
Reverted this.